Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1702,12 +1769,13 @@ LayoutUnit RenderGrid::rowAxisOffsetForChild(const RenderBox& child) const
     LayoutUnit endOfColumn;
     gridAreaPositionForChild(child, ForColumns, startOfColumn, endOfColumn);
     LayoutUnit startPosition = startOfColumn + marginStartForChild(child);
+    LayoutUnit masonryOffset = style().gridMasonryColumns() ? m_masonryLayout.offsetForChild(child) : 0_lu;
```

Comment:
Ditto: should the condition be `style().gridMasonryColumns() && !style().gridMasonryRows()`?

Reply:
Should be fixed now with the helper methods